frontend/src/pages/Registration.js

```diff
@@ -19,67 +28,45 @@ export default function Registration() {
 
     const handleSubmit = event => {
         event.preventDefault()
+        setError()
         createUser(credentials)
-            .catch(error => console.error(error))
+            .then(registeredUser => setRegisteredUser(registeredUser))
+            .catch(error => {setError(error)
+            setLoading(false)})
             .finally(() => setCredentials({credentials: ""}))
-
     }
 
-
+    if(registeredUser){
```

Comment:
Keine Ahnung wo du den registeredUser her holst, aber nimm doch einfach den user aus m AuthContext, wenn da einer da ist, muss er ja auch angemeldet sein 
```suggestion
    if(user){
```